usr.sbin/pkg/CheriBSD.conf

```diff
@@ -0,0 +1,15 @@
+#
+# To disable this repository, instead of modifying or removing this file,
+# create a /usr/local/etc/pkg/repos/CheriBSD.conf file:
+#
+#   mkdir -p /usr/local/etc/pkg/repos
+#   echo "CheriBSD: { enabled: no }" > /usr/local/etc/pkg/repos/CheriBSD.conf
+#
+
+CheriBSD: {
+  url: "pkg+http://pkg.CheriBSD.org/${ABI}/",
```

Comment:
FreeBSD files don't have the trailing slash

Reply:
Fixed-up.